cmd/rm-main.go

```diff
@@ -259,24 +258,23 @@ func removeRecursive(url string, isIncomplete bool, isFake bool, olderThan, newe
 		}
 		urlString := content.URL.Path
 
-		// Skip objects older than --older-than parameter if specified
-		if olderThan != "" && isOlder(content.Time, olderThan) {
-			continue
-		}
+		if !content.Time.IsZero() {
+			// Skip objects older than --older-than parameter if specified
+			if olderThan != "" && isOlder(content.Time, olderThan) {
+				continue
+			}
 
-		// Skip objects newer than --newer-than parameter if specified
-		if newerThan != "" && isNewer(content.Time, newerThan) {
-			continue
+			// Skip objects newer than --newer-than parameter if specified
+			if newerThan != "" && isNewer(content.Time, newerThan) {
+				continue
+			}
 		}
 
-		// list internally mimics recursive directory listing of object prefixes for s3 similar to FS.
-		// The rmMessage needs to be printed only for actual objects being deleted and not prefixes.
-		if !(content.Time.IsZero() && strings.HasSuffix(urlString, string(filepath.Separator))) {
-			printMsg(rmMessage{
-				Key:  targetAlias + urlString,
-				Size: content.Size,
-			})
-		}
+		printMsg(rmMessage{
```

Comment:
Should we print the mssage if `content.Time.IsZero()` ?

Reply:
why should we not?

Reply:
:+1

Reply:
I thought `strings.HasSuffix` is enough to find if the name is a dir or not